.github/workflows/repository-github-super-linter.yml

```diff
@@ -2,39 +2,62 @@
 name: GitHub Super-Linter
 
 on: # yamllint disable-line rule:truthy
-  pull_request:
-    branches:
-      - main
-    types:
-      - edited
-      - opened
-      - reopened
-      - synchronize
-
   schedule:
     - cron: 17 4 * * 1-5
 
-permissions: read-all
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: '${{ github.ref }}-${{ github.workflow }}'
+  cancel-in-progress: true
 
 jobs:
-  super-linter:
+  build:
     name: GitHub Super-Linter
     runs-on: ubuntu-latest
-    permissions:
-      statuses: write
     steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+      - name: Checkout Code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
         with:
+          token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
I don't think this is needed, this is the default https://github.com/actions/checkout/blob/main/action.yml#L24